{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "e0277be0",
   "metadata": {},
   "outputs": [],
   "source": [
    "Example for class 2?"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "131d085d",
   "metadata": {},
   "outputs": [],
   "source": [
    "https://towardsdatascience.com/visualizing-geospatial-data-in-python-e070374fe621"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "c6759da8",
   "metadata": {},
   "outputs": [],
   "source": [
    "conda install -c conda-forge geopandas\n",
    "conda install -c conda-forge geoplot\n",
    "conda install pysal"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "28a4b8c2",
   "metadata": {},
   "outputs": [],
   "source": [
    "Issues here!  Not installing (poss my machine issues?)\n",
    "Looks like needs own conda env.  Is this possible on the noteable servers?"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "61e77252",
   "metadata": {},
   "outputs": [
    {
     "ename": "ModuleNotFoundError",
     "evalue": "No module named 'geoplot'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mModuleNotFoundError\u001b[0m                       Traceback (most recent call last)",
      "Input \u001b[0;32mIn [2]\u001b[0m, in \u001b[0;36m<cell line: 1>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0m \u001b[38;5;28;01mimport\u001b[39;00m \u001b[38;5;21;01mgeoplot\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m \u001b[38;5;21;01mgplt\u001b[39;00m\n\u001b[1;32m      2\u001b[0m \u001b[38;5;28;01mimport\u001b[39;00m \u001b[38;5;21;01mgeopandas\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m \u001b[38;5;21;01mgpd\u001b[39;00m\n\u001b[1;32m      3\u001b[0m \u001b[38;5;28;01mimport\u001b[39;00m \u001b[38;5;21;01mgeoplot\u001b[39;00m\u001b[38;5;21;01m.\u001b[39;00m\u001b[38;5;21;01mcrs\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m \u001b[38;5;21;01mgcrs\u001b[39;00m\n",
      "\u001b[0;31mModuleNotFoundError\u001b[0m: No module named 'geoplot'"
     ]
    }
   ],
   "source": [
    "import geoplot as gplt\n",
    "import geopandas as gpd\n",
    "import geoplot.crs as gcrs\n",
    "import imageio\n",
    "import pandas as pd\n",
    "import pathlib\n",
    "import matplotlib.pyplot as plt\n",
    "import mapclassify as mc\n",
    "import numpy as np"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "126ff2f8",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
